db/migrate/20160720192355_set_page_contents.rb

```diff
@@ -1,24 +1,24 @@
 class SetPageContents < ActiveRecord::Migration
   Page.reset_column_information
-  
+
   home = Page.find_by(name: 'home')
```

Comment:
You could have used `find_or_create_by` here; that way if a page didn't exist, it would create it.